google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -122,8 +122,10 @@ appengine.stop.modules.version.canceled.message=Undeploy Canceled
 appengine.deployment.error.during.staging=Deployment failed due to staging failing with exit code: {0}.
 appengine.deployment.exception.during.staging=Deployment failed due to an exception while staging the project.
 appengine.deployment.error.staging.yaml=The specified app.yaml configuration file does not exist or is not a valid file.
-appengine.deployment.error.staging.dockerfile=The specified Dockerfile configuration file does not exist or is not a valid file.
-appengine.deployment.error.staging.gotosettings=Set a valid file in Module Settings or use another one.
+appengine.deployment.error.staging.dockerfile=There is no Dockerfile in specified directory or it is not a valid file.
+appengine.deployment.error.staging.docker.directory=The specified docker directory does not exist or it is not a valid directory.
+appengine.deployment.error.staging.directory.gotosettings=Set a valid directory in Module Settings or use another one.
```

Comment:
I'd be tempted to use a key label that matched what it's used for instead of what it says.  Maybe appengine.deployment.error.staging.directory.invalid?